ui/i18n/src/main/res/values/strings.xml

```diff
@@ -579,6 +579,9 @@
     <string name="opml_export_label">OPML export</string>
     <string name="html_export_label">HTML export</string>
     <string name="database_export_label">Database export</string>
+    <string name="automatic_database_export_label">Automatic database export</string>
+    <string name="automatic_database_export_summary">Automatically create daily backups of the AntennaPod database</string>
```

Comment:
Maybe here we could add `(only 5 most recent back-ups are kept)`

Reply:
That feels overly detailed, cluttering the settings screen with a wall of text. Signal and WhatsApp have a similar feature and they don't even state that they do the backups daily.

Reply:
> Signal and WhatsApp have a similar feature and they don't even state that they do the backups daily.

In WhatsApp you actually have to choose the frequency (Never, Only when I tap "Back up", Daily, Weekly, Monthly). In Signal it's implied that it's daily, because they note the 'Backup time' without a day.)

For me, the fact that it keeps only 5 most recent is quite relevant. It indicates that I don't need to worry about storage space _and_ I am informed that if my db is f*ed up I need to address it within 5 days, before my correct working database is deleted.
If it is too cluttered, it might be better to find another UX. Both WhatsApp and Signal have full screens dedicated to back-ups.

Reply:
I also agree daily might be a bit much if you can't choose frequency. 
I also feel once a week is a better spot. You have your backup from last month if there really was a problem you didn't notice. Besides once a day could result with backups with no difference apart a play position.

Yet I am sur some people would prefer daily. And I don't think it would clutter settings if you were allowed to choose frequency : off / day / weekly / monthly or just inputting number of days between backups. There is already settings were a dialog is used to select an option.

Even if there is no choice I am glad it's going to be a features. Thanks a lot for this PR!
Less to worry on updates if something goes wrong or if there is any problem with my phone. (I plan to use my backup folder which automatically sync to my PC using syncthing)

Reply:
See #7005